configuration/scripts/cice.run.setup.csh

```diff
@@ -73,10 +75,18 @@ echo " "
 if !(-d \${ICE_LOGDIR}) mkdir -p \${ICE_LOGDIR}
 cp -p \${ICE_RUNLOG_FILE} \${ICE_LOGDIR}
 
-grep ' CICE COMPLETED SUCCESSFULLY' \${ICE_RUNLOG_FILE}
-if ( \$status != 0 ) then
-  echo "CICE run did not complete - see \${ICE_LOGDIR}/\${ICE_RUNLOG_FILE}"
-  echo "\`date\` \${0}: \${ICE_CASENAME} run did NOT complete \${ICE_RUNLOG_FILE}"  >> \${ICE_CASEDIR}/README.case
+if ( \${diagtype} > 0) then
+  grep ' CICE COMPLETED SUCCESSFULLY' \${ICE_RUNLOG_FILE}
+  if ( \$status != 0 ) then
+    echo "CICE run did not complete - see \${ICE_LOGDIR}/\${ICE_RUNLOG_FILE}"
+    echo "\`date\` \${0}: \${ICE_CASENAME} run did NOT complete \${ICE_RUNLOG_FILE}"  >> \${ICE_CASEDIR}/README.case
+    exit -1
+  endif
+else
+  echo "\`date\` \${0}: \${ICE_CASENAME} run completed "
+  echo "CICE NOT run with diag_type='stdout', run status unknown"
```

Comment:
```suggestion
  echo "CICE NOT run with diag_type='stdout', run status unknown - see \${ICE_LOGDIR}/\${ICE_RUNLOG_FILE}"
```

I think in that case it would be nice to also output the path to the log file for easy (\<middle click path\>  + `vi <middle click>`) access